quinn-proto/src/connection/mod.rs

```diff
@@ -1850,6 +1878,49 @@ impl Connection {
         self.streams.queue_max_stream_id(pending);
     }
 
+    /// Modify the number of open paths allowed when multipath is enabled
+    ///
+    /// When reducing the number of concurrent paths this will only affect delaying sending
+    /// new MAX_PATH_ID frames until fewer than this number of paths are possible.  To
+    /// actively reduce paths they must be closed using [`Connection::close_path`], which
+    /// can also be used to close not-yet-opened paths.
+    ///
+    /// If multipath is not enabled in the [`TransportConfig`] this can not enable mulipath
```

Comment:
```suggestion
    /// If multipath is not enabled in the [`TransportConfig`] this can not enable multipath
```

Reply:
also worth noticing that multipath can be enabled but not negotiated, and this will fail as well